src/Nethermind/Nethermind.Mev/Source/CompareMevBundleByMinTimestamp.cs

```diff
@@ -15,23 +15,22 @@
 //  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
 // 
 
-using Nethermind.Core;
-using Nethermind.Int256;
+using System.Collections.Generic;
+using Nethermind.Mev.Data;
 
-namespace Nethermind.Mev
+namespace Nethermind.Mev.Source
 {
-    public class ConstantTailGasPriceCalculator : ITailGasPriceCalculator
+    public class CompareMevBundleByMinTimestamp : IComparer<MevBundle>
     {
```

Comment:
is it needed?

Reply:
yes in sorted pool buckets we want to have it soreted by MinTimestamp